### PR TITLE
Account management

### PR DESCRIPTION
**Pull request by jsjlewis96**
Initial account management work. Won't affect existing functionality

### CODE REVIEW

File: YHP_website/src/main/java/models/Address.java [hunk 1]
```diff
@@ -2,11 +2,109 @@
...
+		return building;
+	}
+
+	/**
+	 * @param building the building to set
+	 */
+	public void setBuilding(String building)
+	{
+		this.building = building;
+	}
```
Comment (Mattihew) [inline YHP_website/src/main/java/models/Address.java]: do we need setters?

File: YHP_website/src/main/java/models/Person.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	// this.id = null;
+	// this.forename = null;
+	// this.surname = null;
+	// }
+
+	public Person(String forename, String surname)
+	{
+		this.id = UUID.randomUUID();
+		this.forename = forename;
+		this.surname = surname;
```
Comment (Mattihew) [inline YHP_website/src/main/java/models/Person.java]: replace with `this(UUID.randomUUID(), forname, surname)`

File: YHP_website/src/main/java/models/Person.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+	}
+
+	public Person(String uuid, String forename, String surname)
```
Comment (Mattihew) [inline YHP_website/src/main/java/models/Person.java]: change  uuid type to `UUID` as database should be able to store values nativley as UUIDs

File: YHP_website/src/main/java/models/User.java [hunk 4]
```diff
@@ -1,12 +1,37 @@
...
+	{
+		super(forename, surname);
+		
+		this.address = address;
+	}
+	
+	public void addAddress(String building, String street, String city_town, String area_code, String country)
+	{
+		this.address = new Address(building, street, city_town, area_code, country);
+	}
```
Comment (Mattihew) [inline YHP_website/src/main/java/models/User.java]: should this object have setters?

**Comment by jsjlewis96** [comment]
Made your changes @Mattihew

**Pull request merged**